### PR TITLE
Feature/fix cesium shadows and add primitives

### PR DESCRIPTION
**Pull request by tomermoshe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,14 @@
 * Added `done` callback to `KeyboardControlService` definitions - now it's possible to know when the event is done.
 * Added internal `IGNORED` state to `KeyboardControlService` - now when action returns `false` it will ignore it until the next event cycle (keydown then keyup)
 * Added CameraService that is provided by `ac-map`. The service exposes the scene's camera and screenSpaceCameraController.
+* Added new components: `ac-billboard-primitive-desc`, `ac-label-primitive-desc`, `ac-polyline-primitive-desc`. The components uses Cesium Primitives for performance efficient drawing of map entities.
```
Comment (eitanfr) [inline CHANGELOG.md]: add small explanation what forRoot does

File: README.md [hunk 2]
```diff
@@ -33,13 +33,17 @@ Check out our [Demo](http://www.angular-cesium.com) that contains small app buil
...
+    		AngularCesiumModule.forRoot()
     	],
     	bootstrap: [AppComponent]
     })
     export class AppModule {
     }
     ```
+    
+###### AngularCesiumModule configuration
++ The main module should be loaded with `.forRoot()` in order to make the module perform enhancements to Cesium, However, the module can be loaded with out calling `forRoot()`.
```
Comment (eitanfr) [inline README.md]: which enhancements? 
`fix cesium shadow bug... `

File: src/services/drawers/entities-drawer/entities-drawer.service.ts [hunk 3]
```diff
@@ -50,12 +50,21 @@ export class EntitiesDrawerService extends BasicDrawerService {
     }
   }
 
+  setDefaultHeight(cesiumProps) {
+    if (this.graphicsType !== GraphicsType.billboard && this.graphicsType !== GraphicsType.label) {
```
Comment (eitanfr) [inline src/services/drawers/entities-drawer/entities-drawer.service.ts]: add to readme

File: CHANGELOG.md (hunk 1 above)
Comment (tomermoshe) [reply CHANGELOG.md]: in readme.md

File: src/services/drawers/entities-drawer/entities-drawer.service.ts (hunk 3 above)
Comment (tomermoshe) [reply src/services/drawers/entities-drawer/entities-drawer.service.ts]: added

File: README.md (hunk 2 above)
Comment (tomermoshe) [reply README.md]: done

**Pull request merged**